djconnectwise/tests/test_sync.py

```diff
@@ -1431,20 +1373,155 @@ def test_sync_multiple_status_batches(self):
 
     def test_delete_stale_tickets(self):
         """Local ticket should be deleted if omitted from sync"""
-        fixture_utils.init_tickets()
-
-        ticket_id = fixtures.API_SERVICE_TICKET['id']
+        ticket_id = self.ticket_fixture['id']
         ticket_qset = Ticket.objects.filter(id=ticket_id)
         self.assertEqual(ticket_qset.count(), 1)
 
-        method_name = 'djconnectwise.api.ServiceAPIClient.get_tickets'
+        method_name = 'djconnectwise.api.TicketAPIMixin.get_tickets'
         mock_call, _patch = mocks.create_mock_call(method_name, [])
-        synchronizer = sync.TicketSynchronizer(full=True)
+        synchronizer = self.sync_class(full=True)
         synchronizer.sync()
         self.assertEqual(ticket_qset.count(), 0)
         _patch.stop()
 
 
+class TestTicketSynchronizer(TestTicketSynchronizerMixin, TestCase):
```

Comment:
I think there should be at least one test for both the service and project syncronizer to make sure that tickets of one type aren't deleted during a sync of the other type of ticket. Let me know if its already in here and I've somehow missed it.